base/strings/annotated.jl

```diff
@@ -505,7 +487,7 @@ end
 function _clear_annotations_in_region!(annotations::Vector{Tuple{UnitRange{Int}, Pair{Symbol, Any}}}, span::UnitRange{Int})
```

Comment:
```suggestion
"""
    _clear_annotations_in_region!(annotations::Vector{Tuple{UnitRange{Int}, Pair{Symbol, Any}}}, span::UnitRange{Int})

remove all annotations that fall within `span` by deleting ranges that are entirely contained, truncating
ranges that partially overlap, and splitting ranges that subsume span
"""
function _clear_annotations_in_region!(annotations::Vector{Tuple{UnitRange{Int}, Pair{Symbol, Any}}}, span::UnitRange{Int})
```
Documenting internals makes it easier for me to review. If this description is right, please accept the suggestion. Otherwise, I'll need to reevaluate my understanding of the function.

Reply:
Mmmm, these functions _should_ all have docstrings. I'll see about adding a commit for this function and others.

Reply:
Docstrings added in `Document the semantics of annotation ordering`